node-client/package.json

```diff
@@ -38,7 +38,7 @@
     "request": "^2.72.0",
     "shelljs": "^0.7.8 ",
     "underscore": "^1.8.3",
-    "websocket": "^1.0.25"
+    "websocket": "^1.0.26"
```

Comment:
The diff on the auto-generated `package-lock.json` is pretty huge, unsure what to do with that, so I'm excluding it for the time being from this PR